scripts/saveCanvas.js

```diff
@@ -0,0 +1,24 @@
+/**
+  Fetch the elements within the canvas then, using _serializeCanvas,
+  create a string represneting that data. Store the resulting string
+  at localStorage.set(storageName).
+
+   @param {string} storageName - a key for a value in localStorage
```

Comment:
All big projects start small -- I strongly recommend using `jsdoc` or SOMETHING at least to remind you and your team what a function is for: http://usejsdoc.org/